.github/pull_request_template.md

```diff
@@ -1,17 +1,20 @@
-## What is this PR doing?
+# What is this PR doing?
 
 
-## How can these changes be manually tested?
+# How can these changes be manually tested?
 
 
-## Does this PR resolve or contribute to any issues?
+# Does this PR resolve or contribute to any issues?
 
 
-## Checklist
+# Checklist
 - [ ] I have manually tested these changes
 - [ ] Post a link to the PR in the group chat
 
-## Guidelines
+# Guidelines
+- Sometimes PRs are a source of conflict, please be mindful of your colleagues
 - If your PR is not ready, mark it as a draft
+- Prefer to seek followup instead of requiring changes, especially if the PR is already a clear net improvement
+- If appropriate, consider implementing followup changes yourself instead of waiting for them from the author
```

Comment:
Unless you have direct permission/acknowledgement from the author, I think direct commits into a PR can at best be rude and at worst lead to conflicts/data loss, such as in some of my workflows where I rebase/force push to limit commits.

Instead, perhaps:
```suggestion
- If appropriate, consider implementing followup changes yourself instead of waiting for them from the author and open a new PR against this one.
```

Reply:
Agreed, suggestion applied